card/lib/card/set/format/abstract_format.rb

```diff
@@ -3,32 +3,104 @@
 class Card
   module Set
     module Format
-      # All Format modules are extended with this module in order to support
-      # the basic format API, including view, layout, and basket definitions
+      # AbstractFormat manages the basic format API, including API to define a {#view}.
+      # Whenever you create a format block in a set module in a {Card::Mod mod}, you
+      # create a format module that is extended with AbstractFormat.
       module AbstractFormat
         include Set::Basket
-        include Set::Format::HamlViews
-        include Set::Format::Wrapper
-
-        VIEW_SETTINGS = %i[cache modal bridge wrap].freeze
+        include ViewOpts
+        include ViewDefinition
+        include HamlViews
+        include Wrapper
+
+        # _Views_ are the primary way that both sharks and monkeys interact with cards.
+        # Sharks select views to use in _nests_.  Monkeys can define and tweak those
+        # views. These docs will introduce the basics of view definition.
+        #
+        # ## Sample view definitions
+        #
+        # Here is a very simple view that just defines a label for the card(its name):
+        #
+        #     view :label do
+        #       card.name
+        #     end
+        #
+        # View definitions can take the following forms:
+        #
+        #     view :viewname[, option_hash][, &block]          # standard
+        #     view :viewname, alias_to_viewname[, option_hash] # aliasing
```

Comment:
the alias API actually changed a little.  I would be willing to get rid of this second option.

Reply:
the changes mean that aliases support other options and templates work without blocks